test/e2e/apimachinery/crd_conversion_webhook.go

```diff
@@ -144,7 +144,7 @@ var _ = SIGDescribe("CustomResourceConversionWebhook [Privileged:ClusterAdmin]",
 		Description: Register a conversion webhook and a custom resource definition. Create a v1 custom
 		resource. Attempts to read it at v2 MUST succeed.
 	*/
-	framework.ConformanceIt("should be able to convert from CR v1 to CR v2", func() {
+	framework.ConformanceIt("Privileged", "should be able to convert from CR v1 to CR v2", func() {
```

Comment:
So a great example here of privileged - we are going to eventually have to add this somewhere else (like an integrator is going to want to have a profile that helps them focus on whether their operator / extension / api-extension works).  So then we need something later on that tags a test into multiple profiles?

Reply:
+1, I think we should change this to an array if we want to support multiple profiles for a test.